src/KeyUnit.cpp

```diff
@@ -90,6 +90,16 @@ void KeyUnit::compileAll()
     }
 }
 
+TKey * KeyUnit::findKey( QString & name ) {
```

Comment:
The iteration that's happening is just an artifact from TAlias, I suspect. I can definitely clean that up, I just wasn't sure if there was a clever reason I didn't understand that it was done that way.